cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMaker2AssemblyOperatorTest.java

```diff
@@ -644,4 +644,86 @@ public void testCreateClusterStatusNotReady(VertxTestContext context) {
             })));
     }
 
+
+    @Test
+    public void testCreateClusterWithZeroReplicas(VertxTestContext context) {
+        ResourceOperatorSupplier supplier = ResourceUtils.supplierWithMocks(true);
+        CrdOperator mockMirrorMaker2Ops = supplier.mirrorMaker2Operator;
+        DeploymentOperator mockDcOps = supplier.deploymentOperations;
+        PodDisruptionBudgetOperator mockPdbOps = supplier.podDisruptionBudgetOperator;
+        ConfigMapOperator mockCmOps = supplier.configMapOperations;
+        ServiceOperator mockServiceOps = supplier.serviceOperations;
+        NetworkPolicyOperator mockNetPolOps = supplier.networkPolicyOperator;
+
+        String clusterCmName = "foo";
```

Comment:
clusterCm is a relic of the past, from when the custom resource was actually a configmap, I planned to remove all references of this at some point, but there are stilla few left over.

```suggestion
        String kmm2Name = "foo";
```